server/sonar-server/src/main/java/org/sonar/server/computation/ComputationContext.java

```diff
@@ -50,11 +53,20 @@ public ComponentDto getProject() {
   }
 
   public BatchReport.Metadata getReportMetadata() {
-    Preconditions.checkState(reportMetadata != null, "Report metadata is available after execution of " + ParseReportStep.class);
+    checkState(reportMetadata != null, "Report metadata is available after execution of " + ParseReportStep.class);
```

Comment:
OK then I would reword it as "Report metadata is available only after execution of ".